pints/_log_likelihoods.py

```diff
@@ -861,6 +862,323 @@ def evaluateS1(self, x):
+            \\ 1 \leq j \leq n_o}} \log \left( \Phi
+            \left(\frac{\text{lower}_{i} - f_j(t_i, \theta)}{\sigma_{i}}
+            \right) \right) \\
+            & + \sum_{\substack{x_{ij} = \text{upper}_{i}, \\1 \leq i
+            \leq n_t, \\ 1 \leq j \leq n_o}}  \log\left(1 -
+            \Phi \left(\frac{\text{upper}_{i} - f_j(t_i, \theta)}{\sigma_{i}}
+            \right) \right)
+        \end{align}
+
+    Extends :class:`ProblemLogLikelihood`.
+
+    Parameters
+    ----------
+    problem
+        A :class:`SingleOutputProblem` or :class:`MultiOutputProblem`.
+    lower
+        The lower limit for censoring.
+    upper
+        The upper limit for censoring.
+    verbose
+        When True, the input data and the values that are censored are printed.
+
+    References
+    ----------
+    .. [3] Beal, S. L. (2001). Ways to fit a PK model with some data below the
+           quantification limit. Journal of pharmacokinetics and
+           pharmacodynamics 28, pp. 481-504.
+
+    """
+
+    def __init__(self, problem, lower=None, upper=None, verbose=True):
+        super(CensoredGaussianLogLikelihood, self).__init__(problem)
+
+        # Get number of times, number of outputs
+        self._nt = len(self._times)
+        self._no = problem.n_outputs()
+
+        # Add parameters to problem
+        self._n_parameters = problem.n_parameters() + self._no
+        a = lower
+        if a is not None:
+            if np.isscalar(a):
+                a = np.ones(self._no) * float(a)
+            else:
+                a = pints.vector(a)
+                if len(a) != self._no:
+                    raise ValueError(
+                        'Lower limit must be a ' +
+                        ' scalar or a vector of length n_outputs.')
+        b = upper
+        if b is not None:
+            if np.isscalar(b):
+                b = np.ones(self._no) * float(b)
+            else:
+                b = pints.vector(b)
+                if len(b) != self._no:
+                    raise ValueError(
+                        'Upper limit must be a ' +
+                        ' scalar or a vector of length n_outputs.')
+
+        if (a is not None) and (b is not None):
+            diff = b - a
+            if np.any(diff <= 0):
+                raise ValueError('Upper limit ' +
+                                 'must exceed lower limit.')
+        self._a = a
+        self._b = b
+
+        # Define the condition for whether a point is not censored
+        if (self._a is not None) and (self._b is not None):
+            self._condition = (self._a < self._values
+                               ) & (self._values < self._b)
+        elif (self._a is not None) and (self._b is None):
+            self._condition = self._a < self._values
+        elif (self._a is None) and (self._b is not None):
+            self._condition = self._values < self._b
+        else:
+            raise ValueError("Not a censored likelihood")
```

Comment:
With the change above you can simply this logic and don't actually need to raise the ValueError anymore:

self.condition = (self._values >= self._a) & (self._values <= self._b)